clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -73,12 +83,13 @@ public boolean equals(Object o) {
             && leaderEpoch == that.leaderEpoch
             && highWatermark == that.highWatermark
             && Objects.equals(voters, that.voters)
-            && Objects.equals(observers, that.observers);
+            && Objects.equals(observers, that.observers)
+            && Objects.equals(nodes, that.nodes);
```

Comment:
Will the equals work as expected if we don't override equals method?

Reply:
Node#equals, Node#hashCode added.